web-server/src/routes/v1/users.ts

```diff
@@ -1,6 +1,6 @@
 import { Router } from 'express';
 
-import { list, show, edit, destroy } from 'controllers/users';
+import { list, show, edit, destroy, update } from 'controllers/users';
```

Comment:
Can you just use `edit` apparently there's one already? :D

I mean, you can replace this `edit` function entirely with the contents of your `update` function. They seem to serve the same purpose.